src/SignalR/server/Core/src/HubInvocationContext.cs

```diff
@@ -29,6 +41,11 @@ public HubInvocationContext(HubCallerContext context, string hubMethodName, obje
         /// </summary>
         public HubCallerContext Context { get; }
 
+        /// <summary>
+        /// Gets the Hub name.
+        /// </summary>
+        public string HubName { get; }
```

Comment:
```suggestion
        public Type Hub { get; }
```

I should have said something about this earlier, but I think we should expose the Hub type instead of just the Hub name. That way developers can get the fully qualified name and other details. Thanks.

Reply:
🤔 good point, but how do we get the name of the hub with `HubType` prop?

Reply:
[Type.FullName](https://docs.microsoft.com/dotnet/api/system.type.fullname?view=netframework-4.8) or [Type.Name](https://docs.microsoft.com/dotnet/api/system.type.name?view=netstandard-1.6&viewFallbackFrom=netframework-4.8)

Reply:
make sense, @BrennanConroy so should I change the `HubName` to `HubType` and revert my previous changes?

Reply:
> should I change the `HubName` to `HubType`

Yes please!

Reply:
@Potapy4 I think we're good to merge this PR once this change is in!

Reply:
Oh, and since this is an API change, could you update the reference assemblies: https://github.com/aspnet/AspNetCore/blob/master/docs/ReferenceAssemblies.md#when-changing-public-api

Should just need to run `dotnet msbuild /t:GenerateReferenceSource` from the signalr/server/core/src/ folder.

Reply:
@davidfowl @anurse Do we need any sort of API review before merging this?

Reply:
> @Potapy4 I think we're good to merge this PR once this change is in!

Sorry for the delay, I will do it soon! Quick question: if we change it to `Type HubType` - should we still pass `hubname` to the constructor?

Reply:
We would want to pass `hubType` in the constructor instead :)